routes/index.js

```diff
@@ -4,7 +4,8 @@ const users = require('./users');
 const semesters = require('./semesters');
 const modules = require('./modules');
 const courses = require('./courses');
-const { ensureAuthenticated, ensureIsAdmin } = require('../auth/middleware');
+
+const { ensureAuthenticated, ensureIsAdmin, ensureProfessor } = require('../auth/middleware');
```

Comment:
```suggestion
const { ensureAuthenticated, ensureIsAdmin, ensureProfessor } = require('../auth/middleware');
```

Can you revert the changes here? I am going to refactor the permissions